appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java

```diff
@@ -95,15 +103,38 @@ public void onMouseMove(MouseMoveEvent event) {
         clientY = event.getClientY();
       }
     });
+    
+    
     assetList.addSelectionHandler(new SelectionHandler<TreeItem>() {
+      private int clickCount = 0;
       @Override
       public void onSelection(SelectionEvent<TreeItem> event) {
```

Comment:
You will need to make the parameter here `final` for this to compile.

Reply:
fixed